### PR TITLE
Fix partially visible maneuver problem

### PR DESCRIPTION
**Pull request by ShanMa1991**
### Description
This pr is to fix the partially visible maneuver problems during navigation session when multiple waypoints are selected.

### Implementation
By changing the sequence of laying addition to fix the partially visible problem caused by layer overlapping.

### Screenshots or Gifs
maneuver_fix

### CODE REVIEW

File: Sources/MapboxNavigation/NavigationMapView.swift [hunk 1]
```diff
@@ -518,9 +518,9 @@ open class NavigationMapView: UIView {
             var parentLayer: String? = nil
             let identifiers = [
                 IdentifierString.arrow,
-                IdentifierString.arrowSymbol,
```
Comment (MaximAlien) [inline Sources/MapboxNavigation/NavigationMapView.swift:521]: Original sequence of layer was working correctly in Navigation SDK `1.x`, why this change is required in `2.x`?

Comment (ShanMa1991) [reply Sources/MapboxNavigation/NavigationMapView.swift:521]: In Carbon if we're using the 1.x , the adding layer sequence would cause the maneuvers not shown at all. The new sequence of the adding identifiers is following the layer addition and overlapping sequence.

**Review by MaximAlien — APPROVED** [review]
